crates/snippet_provider/src/format.rs

```diff
@@ -36,9 +70,14 @@ impl std::fmt::Display for ListOrDirect {
     }
 }
 
-#[derive(Deserialize)]
+#[derive(Deserialize, JsonSchema)]
 pub(crate) struct VSCodeSnippet {
+    /// The snippet prefix used for intellisense.
```

Comment:
We shouldn't be using `intellisense` name here. An alternative could be `The snippet prefix used to decide whether a completion menu should be shown` or something along the lines.